python/tests/conftest.py

```diff
@@ -238,6 +238,26 @@ def arctic_library(arctic_client, lib_name):
     return arctic_client.create_library(lib_name)
 
 
+@pytest.fixture(
+    scope="function",
+    params=[
```

Comment:
Why is this parameterized? Isn't just LMDB fine? Isn't there a similar fixture already?

Reply:
I was trying to create a library fixture similar to existing `def basic_store()`
It is used by `test_write_parallel_sort_merge` which tests parallelism. So I think it's safer to test multiple storages.